frontend/src/assets/svgs/wheel-background/back-button.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="32" height="32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Same story:

- Needs to be run through the optimizer
- Remove `width`, `height`
- Set `fill` to `currentColor`
- Add `color` in component